### DISCUSSION TITLE
is there a way to turn off stream parquet write

### DISCUSSION THREAD

**Discussion by djouallah** [duckdb/duckdb#10960] [Q&A]
since duckdb 0.10, it seems duckdb start writing parquet as a stream, which is great if you write to a local SSD but it is generate errors if you write to a remote storage like Fabric onelake, is there an option to turn off streaming and write the parquet files only once.
currently the workaround is to use 0.9.2

**Comment by Mause** [comment]
What error are you seeing?

**Reply by djouallah** [reply]
the errors are not from duckdb, which by the way works fine in my laptop, but the errors are from spark runtime, so java stuff
